front/app/api/map/communes/route.tsx

```diff
@@ -6,25 +6,27 @@ export async function GET(request: NextRequest) {
   try {
     const searchParams = request.nextUrl.searchParams;
     const codes = searchParams.getAll('codes');
+    const year = searchParams.get('year') || '2024';
```

Comment:
```suggestion
    const year = searchParams.get('year') || '2023';
```
Les derniers scores de transparence ont été calculés sur 2023 et pas 2024. C'est pour ça qu'on a que des E sur les régions et départements (ce qui n'est pas le cas en 2023 même si ça reste pas élevé...)